Samples/ExtendedControlAndMetadata/EyeGazeAndBackgroundSegmentation/EyeGazeAndBackgroundSegmentation/EyeGazeAndBackgroundSegmentation.csproj

```diff
@@ -0,0 +1,183 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">x86</Platform>
+    <ProjectGuid>{6B0DAB20-BB5C-4738-B8F0-102EE1E2EC6F}</ProjectGuid>
+    <OutputType>AppContainerExe</OutputType>
+    <AppDesignerFolder>Properties</AppDesignerFolder>
+    <RootNamespace>EyeGazeAndBackgroundSegmentation</RootNamespace>
+    <AssemblyName>EyeGazeAndBackgroundSegmentation</AssemblyName>
+    <DefaultLanguage>en-US</DefaultLanguage>
+    <TargetPlatformIdentifier>UAP</TargetPlatformIdentifier>
+    <TargetPlatformVersion Condition=" '$(TargetPlatformVersion)' == '' ">10.0.21364.0</TargetPlatformVersion>
+    <TargetPlatformMinVersion>10.0.19041.0</TargetPlatformMinVersion>

```

Comment:
>19041 [](http://example.com/codeflow?start=35&length=5)

Does this need to be updated?

Reply:
20348 does not have a UWP target. I can set as minimum version.. I plan on updating to whatever sdk is Next. I'll make 29041 as minimal one since these APIs are accessible there